### ISSUE TITLE
(Most Recent Nightly Build) REFramework Menu HUGE In-Game

### ISSUE THREAD

**Issue by NateHevens** [praydog/REFramework#1078]
I'm using the most recent [nightly build](https://github.com/praydog/REFramework-nightly/releases) in Dragon's Dogma 2. The menu you get when pressing Insert looks normal-sized until I actually get into the game, and then suddenly it's massive (as you can see in the pic). I tried to edit text size and it caused the game to crash,
![Dragon's Dogma 2 6_8_2024 12_07_05 AM](https://github.com/praydog/REFramework/assets/36975652/eb670e95-87b2-4220-a85a-a54c04b02acf)

**Comment by praydog** [comment]
One of your scripts is doing this and not resetting the font back to normal. Dig through your mods and disable the offending one.

**Comment by NateHevens** [comment]
@praydog Is there any way to determine which one exactly? Or just do I have to go one-by-one?

**Comment by praydog** [comment]
No, just disable all of them and turn them on one by one until it breaks. It might not get set back to normal without completely closing the game, just be aware.

**Comment by NateHevens** [comment]
Got it. Thanks.

**Comment by derickso** [comment]
Which one was it? I would like to make mine larger.
